api/envoy/config/route/v3/route_components.proto

```diff
@@ -317,6 +317,21 @@ message Route {
   // If set, the bytes actually buffered will be the minimum value of this and the
   // listener per_connection_buffer_limit_bytes.
   google.protobuf.UInt32Value per_request_buffer_limit_bytes = 16;
+
+  // The human readable prefix to use when emitting statistics for this endpoint.
+  // The statistics are rooted at vhost.<virtual host name>.route.<stat_prefix>.
+  // This should be set for highly critical
+  // endpoints that one wishes to get “perfect” statistics on.
```

Comment:
can you define "perfect" here?

Reply:
It just means we get perfect observability per route  - instead of combined stats for all routes. TBH, this part was taken from virtual clusters docs. Any suggestions to improve?